source/includes/administration/_notifications.md

```diff
@@ -0,0 +1,364 @@
+            }
+        ],
+        "deprecatedAt": "2020-11-03T20:57:00.000Z",
+        "sticky": true,
+        "id": "b7c6d7ca-6fef-406b-a911-bba873a99773",
+        "category": {
+            "organizationId": "1384d793-dae5-441c-8cc2-e78168c94018",
+            "createdAt": "2020-10-29T15:01:44.000Z",
+            "translations": [
+                {
+                    "language": "en",
+                    "id": "4840af1e-1d74-4b4f-bdda-6e04f4569142",
+                    "text": "system-category",
+                    "type": "title",
+                    "version": 1
+                },
+                {
+                    "language": "fr",
+                    "id": "0bc8f49c-8f65-4319-be57-ad70cdfe144d",
+                    "text": "en français",
+                    "type": "title",
+                    "version": 1
+                }
+            ],
+            "icon": "fa fa-envelope",
+            "id": "2569f2cc-f1d7-4664-8439-173fedfab1c3",
+            "version": 1,
+            "updatedAt": "2020-10-29T19:28:22.000Z"
+        },
+        "splash": false,
+        "updatedAt": "2020-10-29T19:27:34.462Z",
+        "status": "published"
+    }
+}
+```
+Get a specific notification.
+
+Attributes | &nbsp;
+---------- | -----------
+`id`<br/>*UUID* | The id of the notification.
+`authorId`<br/>*UUID* | The id of the notification's author.
+`authorFirstName`<br/>*string* | The notification's author's first name.
+`authorLastName`<br/>*string* | The notification's author's last name.
+`splash`<br/>*boolean* | When set to true, the notification will be displayed in the home screen of CloudMC until it reaches its deprecated date. Only one of splash and sticky can be set to true.
+`sticky`<br/>*boolean* | When set to true, the notification will be displayed upon logging into CloudMC. Only one of splash and sticky can be set to true.
+`status`<br/>*string* | The status of the notification.
+`createdAt`<br/>*string* | The date on which the notification was created.
+`updatedAt`<br/>*string* | The date on which the notification was updated.
+`publishedAt`<br/>*string* | The date on which the notification was published.
+`depricatedAt`<br/>*string* | The date on which the notification will be deprecated.
+`version`<br/>*integer* | The notification's version.
+`category`<br/>*Array[[Notification Category](#administration-notification-categories)]* | The notification category that the notification belongs to.<br/>*includes*: `id`, `organizationId`, `createdAt`, `updatedAt`, `translations`, `icon`, `version`
+`translations`<br/>*Array[Object]* | The translations of the notification category's title.
+`translations.language`<br/>*string* | The language of that the translation is in.
+`translations.text`<br/>*string* | The content of the notification.
+`translations.type`<br/>*string* | The content that we are translating. (Must be one of: title, body, description, url slug)
+`translations.id`<br/>*UUID* | The translation's id.
```

Comment:
apply the same comment as the list

Reply:
40448fe47ff6582d909c3046be19ca1060e34bf7